lib/po.js

```diff
@@ -32,9 +33,8 @@ PO.prototype.toString = function () {
     lines.push('msgid ""');
     lines.push('msgstr ""');
 
-    var keys = Object.keys(this.headers);
     var self = this;
-    keys.forEach(function (key) {
+    this.headerOrder.forEach(function (key) {
```

Comment:
What happens if I manually do `po.headers["test"] = "123"`.

Not sure if that will still get serialized?

Reply:
Fixed.